### PR TITLE
Renders dashboard page for when user successfully logs in

### PR DESCRIPTION
**Pull request by holgertidemand**
#### Description
Changes proposed in this pull request:

* Adds dashboard page 
* User can see links to create and join group
#### What I have learned working on this feature: 
Learned how to render a new view instead of using another controller

#### Screenshots:
screen shot 2018-02-28 at 13 57 47

### CODE REVIEW

**Review by magnus-thor — DISMISSED** [review]
Looks good.

File: features/step_definitions/authenticated_user_steps.rb [hunk 1]
```diff
@@ -28,3 +24,14 @@
 Then("show me the page") do
   save_and_open_page
 end
+
+Then("I should be on the Sign in page") do
```
Comment (diraulo) [inline features/step_definitions/authenticated_user_steps.rb]: This is all fine, but... You will end up with a lot of steps that are doing exactly the same thing. You can refactor this to make it more dynamic. You could have a step like `I should be on the {string} page` and it's implementation would look something like

```ruby
Then("I should be on the {string} page") do |page_name|
  if page_name == 'Sign in'
    path = new_user_session_path
  end

  expect(page.current_path).to eq path
end
```

To make this even better you could extract the page path selection into a method

```ruby
def page_path_from(page_name)
  if page_name == 'Sign in'
    new_user_session_path
  elsif page_name == 'Sign up'
    user_registration_path
  else
    raise "#{page_name} is not defined"
  end
end
```

And you replace your expectation above with

```ruby
Then("I should be on the {string} page") do |page_name|
  expect(page.current_path).to eq page_path_from(page_name)
end
```

File: features/user_authentication.feature [hunk 2]
```diff
@@ -3,12 +3,11 @@ Feature: User is authenticated for both sign up and login.
...
   We need to be able to sign up or log in
 
-
   Background:
     Given the following user exists
-      |  email    | password    | password_confirmation |
+      | email    | password    | password_confirmation |
       | jf@ca.com | password123 | password123           |
-    And I am on the "Landing" page
+    And I am on the Landing page
```
Comment (diraulo) [inline features/user_authentication.feature]: So if you implement the above suggestion you can keep this step the way it was before

**Review by diraulo — COMMENTED** [review]
Looks alright in general. But check the comment i just left in

**Review by diraulo — APPROVED** [review]

**Pull request merged**